readme.md

```diff
@@ -233,13 +224,39 @@ If wish to install pre-commit as a pre-commit git hook, you can run (optional):
 pre-commit install
 ```
 
-### Security settings (`security_settings`)
+
+## Sass compilation (`sass_bootstrap`)
+
+Use this feature to enable Sass processing and Bootstrap styling.
+
+While you can just include Bootstrap's styling/js via a CDN, using this feature allows you to customize Bootstrap to the
+style guide of your project, as well as define custom styling in a cleaner and more maintainable way (compared to plain
+CSS). The Bootstrap part of this integration could be swapped out for any other frontend styling framework that also
+uses Sass, but there really is no reason to write vanilla CSS.
+
+In local development, you can simply write scss files and include them using `sass_tags` and your stylesheets should
+automatically recompile in reload. This also works seamlessly with `collectstatic` for deploys.
+
+Note: If you aren't already using npm to install bootstrap, you can alternatively clone the contents of Bootstrap's sass
+files directly into your static directory and change your references to point there. There is currently no good way to
+install Bootstrap source code using just python.
+
+### Production notes
+
+In development, `.scss` files are compiled on the fly. However, when deploying, these files must be manually generated
+using `python manage.py compilescss`. Also note that if your styles folder is in a directory that's collected with
+`collectstatic`, you should add the `--ignore *.scss` flag to avoid exposing the raw `.scss` files as staticfiles.
+
+
+## Security settings (`security_settings`)
 These are the recommended security settings. [Explanations for all Django settings can be found here](https://docs.djangoproject.com/en/3.2/ref/settings/). Please pay particular note to what are appropriate cookie and subdomain settings for your application.
 
-### Sentry integration (`sentry`)
+## Sentry integration (`sentry`)
+
 
+## User Action Tracking (`user_action_tracking`)
```

Comment:
Can we add a brief description of what this is used for and some basic details like `USER_TRACKING_EXEMPT_ROUTES` is a list of route names and not route urls?